getting-started/kubernetes/hardway/test-networking.md

```diff
@@ -11,13 +11,13 @@ In this lab we will test the {{site.prodname}} cluster to demonstrate networking
 Create three busybox instances
 
 ```bash
-kubectl run pingtest --image=busybox --replicas=3 -- sleep infinity
+kubectl create deployment pingtest --image=busybox --replicas=3 -- sleep infinity
 ```
 
 Check their IP addresses
 
 ```bash
-kubectl get pod -l run=pingtest -o wide
+kubectl get pods --field-selector=app=pingtest -o wide
```

Comment:
I don't think we want to use a field selector here - it should be a normal label selector (e.g., `-l`) in order to match labels.